infra/lib/eduace-backend.ts

```diff
@@ -0,0 +1,38 @@
+import {Bucket} from "@aws-cdk/aws-s3";
+import {CfnOutput, Construct} from "@aws-cdk/core";
+import {LambdaRestApi} from "@aws-cdk/aws-apigateway";
+import {Code, Function as LambdaFunction} from "@aws-cdk/aws-lambda";
+import {Runtime} from "@aws-cdk/aws-lambda";
+
+export class EduaceBackend extends Construct {
+    constructor(scope: Construct, id: string) {
+        super(scope, id);
+
+        const sourceCodeBucket = new Bucket(this, "EduaceBackendLambdaStore")
```

Comment:
How exactly does this get the python code?

Reply:
Oh it doesn't, it's just literally an empty bucket with the name "EduaceBackendLambdaStore"